app/dashboards/category_dashboard.rb

```diff
@@ -0,0 +1,58 @@
+ require "administrate/base_dashboard"
+
+class CategoryDashboard < Administrate::BaseDashboard
+  # ATTRIBUTE_TYPES
+  # a hash that describes the type of each of the model's fields.
+  #
+  # Each different type represents an Administrate::Field object,
+  # which determines how the attribute is displayed
+  # on pages throughout the dashboard.
+  ATTRIBUTE_TYPES = {
+      id: Field::Number,
+      name: Field::String,
+      short_summary: Field::String,
+      summary: Field::String,
+      parent: Field::BelongsTo.with_options(class_name: "Category"),
+      children: Field::HasMany.with_options(class_name: "Category"),
+  }.freeze
+
+  # COLLECTION_ATTRIBUTES
+  # an array of attributes that will be displayed on the model's index page.
+  #
+  # By default, it's limited to four items to reduce clutter on index pages.
+  # Feel free to add, remove, or rearrange items.
+  COLLECTION_ATTRIBUTES = [
+      :id,
+      :name,
+      :parent
+  ].freeze
+
+  # SHOW_PAGE_ATTRIBUTES
+  # an array of attributes that will be displayed on the model's show page.
+  SHOW_PAGE_ATTRIBUTES = [
```

Comment:
Handy to also show `:slug`. We also usually include `created_at` and `updated_at`.